### PR TITLE
Added functionality to deploy MongoDB docker containers and updated maven sure-fire plugin

### LINKED ISSUE #5: Add support to deploy MongoDB
https://hub.docker.com/_/mongo

Add support to deploy all available versions of MongoDB.

### PR DESCRIPTION
**Pull request by the-c0d3br34k3r**
Closes #5.
- Currently the deployment of the latest MongoDB docker image is supported.
- Updated Maven sure-fire plugin from 3.0.0-M3 to 3.0.0-M5.

### CODE REVIEW

File: src/test/java/DockerUtilityTest.java [hunk 1]
```diff
@@ -19,28 +26,41 @@ public static void setUp() throws InterruptedException {
...
                 DockerImage.MICROSOFT_SQL_SERVER_2019_LATEST,
-                DockerImage.MYSQL_LATEST
+                DockerImage.MYSQL_LATEST,
+                DockerImage.MONGODB_LATEST
         );
+
         // containersToRun should be a subset of containersToCreate, as only if the container is created can it be run.
+        runningContainers = new HashMap<>();
         containersToRun = Arrays.asList(
                 DockerImage.MICROSOFT_SQL_SERVER_2019_LATEST,
```
Comment (abprash) [inline src/test/java/DockerUtilityTest.java:44]: Little out of the context, why are we not running all the created containers?

Comment (abprash) [reply src/test/java/DockerUtilityTest.java:44]: nvm, got it, this is for the else block below.

File: src/test/java/DockerUtilityTest.java [hunk 2]
```diff
@@ -19,28 +26,41 @@ public static void setUp() throws InterruptedException {
...
         dockerUtility = new DockerUtility();
         for (DockerImage imageToPull : imagesToPull) {
             dockerUtility.pullImage(imageToPull);
         }
         for (DockerImage containerToCreate : containersToCreate) {
-            dockerUtility.createContainer(containerToCreate, containerToCreate.toString(), true);
-        }
-        for (DockerImage containerToRun : containersToRun) {
-            dockerUtility.createAndRunContainer(containerToRun, containerToRun.toString(), true);
+            String containerName = containerToCreate.name() + "_" + containerToCreate.toString() + "_" + Instant.now().getEpochSecond();
```
Comment (abprash) [inline src/test/java/DockerUtilityTest.java:54]: do you think it's better to name the container, in a more human readable way -
name_name_YYYY_MM_dd_hh_mm_ss vs name_name_<long number>?

Comment (the-c0d3br34k3r) [reply src/test/java/DockerUtilityTest.java:54]: Honestly, doesn't matter. The usage of epoch was to ensure unique names, the actual time is not useful at that point considering we have logging for when we start/stop containers.

Comment (abprash) [reply src/test/java/DockerUtilityTest.java:54]: 👍

**Review by abprash — APPROVED** [review]

**Pull request merged**